src/components/ErrorSummary.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { ErrorInfo } from '../types';
+
+export type ErrorSummaryInfo = {
+    errors: ErrorInfo[];
+    errorHref?: string;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
If we do that, we get "Type '({ errors, errorHref }: ErrorSummaryInfo) => Element | null' is missing the following properties from type 'ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)> | null) | (new (props: any) => Component<any, any, any>)>': type, props, keyts(2739)"

Reply:
Hmm strange, I don't get an error when I amend it. Where are you putting the type?
I've used this for line 10 and it seems happy `const ErrorSummary = ({ errors, errorHref }: ErrorSummaryInfo): null | ReactElement => {`

Reply:
Ah no that works - it was because it had :any after const ErrorSummary. Removing that makes it happy (i think its an implicit any rather than explicit now).